### PR TITLE
refactor(x/bank): remove message events including `sender` attribute whose information is already present in the relevant events (backport cosmos/cosmos#17273)

### PR DESCRIPTION
**Pull request by 0Tech**
## Description

This patch would remove duplicate `message` events in x/bank keeper.

You may refer to the PR in the upstream.

## Checklist:

- [x] I followed the [contributing guidelines](https://github.com/Finschia/finschia-sdk/blob/main/CONTRIBUTING.md) and [code of conduct](https://github.com/Finschia/finschia-sdk/blob/main/CODE_OF_CONDUCT.md).
- [x] I have added a relevant changelog to `CHANGELOG.md`
- [x] I have added tests to cover my changes.
- [ ] I have updated the documentation accordingly.
- [ ] I have updated API documentation `client/docs/swagger-ui/swagger.yaml`

### CODE REVIEW

**Comment by 0Tech** [comment]
Should rebase it after #1063 merged.

File: CHANGELOG.md [hunk 1]
```diff
@@ -46,6 +46,8 @@ Ref: https://keepachangelog.com/en/1.0.0/
 ### Removed
 
 ### Breaking Changes
+* (refactor) [\#1063](https://github.com/Finschia/finschia-sdk/pull/1063) Automate EventTypeMessage inclusion in every message execution (backport cosmos/cosmos-sdk#13532)
+* (x/bank) [#17273](https://github.com/cosmos/cosmos-sdk/pull/17273) Remove message events including `sender` attribute whose information is already present in the relevant events
```
Comment (0Tech) [inline CHANGELOG.md:50]:
```suggestion
* (x/bank) [#1066](https://github.com/Finschia/finschia-sdk/pull/1066) Remove message events including `sender` attribute whose information is already present in the relevant events
```

**Review by zemyblue — COMMENTED** [review]
I think the `AttributeKeySender` of bank module is no need. 
https://github.com/Finschia/finschia-sdk/blob/2afab695dd5961c093dd0e16727eda1535a66cac/x/bank/types/events.go#L12

File: x/authz/keeper/keeper_test.go [hunk 2]
```diff
@@ -231,13 +231,12 @@ func (s *TestSuite) TestDispatchedEvents() {
 	events := s.ctx.EventManager().Events()
 
 	// get last 5 events (events that occur *after* the grant)
```
Comment (jaeseung-bae) [inline x/authz/keeper/keeper_test.go:233]: comment may needs update(  `// get last 5 events` to `// get last 4 events` )

**Comment by 0Tech** [comment]
> I think the `AttributeKeySender` of bank module is no need.
> 
> https://github.com/Finschia/finschia-sdk/blob/2afab695dd5961c093dd0e16727eda1535a66cac/x/bank/types/events.go#L12

The attribute is still in use, e.g. in `transfer` event.

File: x/authz/keeper/keeper_test.go (hunk 2 above)
Comment (0Tech) [inline x/authz/keeper/keeper_test.go:233]:
```suggestion
	// get last 3 events (events that occur *after* the grant)
```

File: x/authz/keeper/keeper_test.go [hunk 3]
```diff
@@ -231,13 +231,12 @@ func (s *TestSuite) TestDispatchedEvents() {
...
-	events = events[len(events)-5:]
+	events = events[len(events)-4:]
```
Comment (0Tech) [inline x/authz/keeper/keeper_test.go:234]:
```suggestion
	events = events[len(events)-3:]
```

**Review by 0Tech — COMMENTED** [review]
Update the unit test.

**Review by jaeseung-bae — APPROVED** [review]

**Pull request merged**